### PR TITLE
UI Fixes/Tweaks

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Made a series of UI fixes that make the application look better. Changes made:

- Modified related buttons to have the same look and feel
- Modified buttons to be of smaller text size
- Added the root static directory to the settings file
- Added a command in the initialize script to run collectstatic at startup

 
Screen Shot 2023-03-31 at 2 49 26 PM

### CODE REVIEW

File: saas_app/submit_request/forms.py [hunk 1]
```diff
@@ -30,15 +30,13 @@ def __init__(self, *args, **kwargs):
...
-            Submit("submit", _("Submit"), css_class="btn btn-primary")
-        )
+        self.helper.add_input(Submit("submit", _("Submit")))
         self.helper.add_input(
             Button(
                 "cancel",
                 _("Cancel"),
                 css_id="submit",
-                css_class="btn btn-primary btn-lg",
+                css_class="btn btn-primary btn-med",
```
Comment (gcharest) [inline saas_app/submit_request/forms.py:39]: It's the first time I see a `btn-med` class, is that custom or part of crispy?

**Review by gcharest — APPROVED** [review]
Non-blocking comment:
- Overall the tweaks make for a more consistent UI
- There are a few little hits with Axe for a11y, mainly on the buttons and navlinks at the top due to text contrast not being high enough on their background.
   - For the buttons, I think the default class gets overwritten by the plugins somehow

File: saas_app/submit_request/forms.py (hunk 1 above)
Comment (sylviamclaughlin) [reply saas_app/submit_request/forms.py:39]: You know what, you are totally right! In general btn-lg, btn-sm are part of bootstrap but btn-med is not (or at least I could not find it now). Good catch! I will play around with this to actually use a button class that is valid.

Comment (sylviamclaughlin) [reply saas_app/submit_request/forms.py]: I removed btn-med (no idea where I got it from) and then I made the buttons have a consistent look. As you pointed out, it meets better accessiblity standards. 

Screen Shot 2023-04-03 at 5 27 00 PM

**Pull request merged**